src/main.rs

```diff
@@ -61,19 +63,21 @@ async fn db_connection() -> Client {
 /// Establishes connection to Dapr.
 ///
 /// Adds AppCallbackService which defines pub/sub interaction with Dapr.
-async fn dapr_connection() {
-    let addr = "[::]:50006".parse().unwrap();
-
-    let callback_service = AppCallbackService::default();
+async fn dapr_connection(db_client: Database) {
+    let addr = "[::]:50051".parse().unwrap();
```

Comment:
Why did the address change?

Reply:
This was for debugging purposes and the same change is reflected in the respective Docker Compose file, the changes there are not yet pushed.